examples/ex15.yml

```diff
@@ -0,0 +1,47 @@
+connectors:
+  X1:
+    type: Molex KK 254 # more information
+    subtype: female
+    pinlabels: [GND, VCC, RX, TX, GND, VCC, GND] # pincount is implicit in pinout
+    shorts:
+      - SH1: [1, 5, 7]
+      - SH2: [2, 6]
```

Comment:
Using a list here is fine. Does not match the syntax description though.

Reply:
Syntax description is yet to be updated.